elvis.config

```diff
@@ -8,7 +8,10 @@
        ruleset => erl_files,
        rules => [{elvis_style, line_length, #{limit => 100,
                                               skip_comments => false}},
-		 {elvis_style, dont_repeat_yourself, #{min_complexity => 20}}
+                 {elvis_style,
+                  variable_naming_convention,
+                  #{regex => "^_{0,2}([A-Z][0-9a-zA-Z]*)$"}},
```

Comment:
You mean single underscore, or double? Elvis by default allows single underscore, we allow double underscore for reasons already mentioned.